test/pathforaSpec.js

```diff
@@ -1083,7 +1083,7 @@ describe('API', function () {
     var widget = $('#' + form.id);
     expect(widget.length).toBe(1);
   });
-  
+
   it('should open site gating widget when the cookie is not set', function (done) {
     var gate = new window.pathfora.SiteGate({
       header: 'Blocking Widget',
```

Comment:
I think these should be `headline` now. Shouldn't cause any problems in the tests though.